Sources/TuistKit/Mappers/Graph/StaticXCFrameworkModuleMapGraphMapper.swift

```diff
@@ -0,0 +1,182 @@
+                        case let .xcframework(xcframework):
+                            return xcframework
+                        default:
+                            return nil
+                        }
+                    }
+
+                guard !staticObjcXCFrameworksLinkedByDynamicXCFrameworkDependencies.isEmpty else { continue }
+
+                sideEffects += try self.sideEffects(
+                    for: staticObjcXCFrameworksLinkedByDynamicXCFrameworkDependencies,
+                    derivedDirectory: derivedDirectory
+                )
+
+                settingsToPropagate[.target(name: target.name, path: project.path)] = [
+                    "OTHER_SWIFT_FLAGS": .array(
+                        staticObjcXCFrameworksLinkedByDynamicXCFrameworkDependencies.flatMap { xcframework -> [String] in
+                            [
+                                "-Xcc",
+                                moduleMapFlag(
+                                    for: xcframework,
+                                    derivedDirectory: derivedDirectory,
+                                    project: project
+                                ),
+                            ]
+                        }
+                    ),
+                    "OTHER_C_FLAGS": .array(
+                        staticObjcXCFrameworksLinkedByDynamicXCFrameworkDependencies.flatMap { xcframework -> [String] in
+                            [
+                                moduleMapFlag(
+                                    for: xcframework,
+                                    derivedDirectory: derivedDirectory,
+                                    project: project
+                                ),
+                            ]
+                        }
+                    ),
+                    "HEADER_SEARCH_PATHS": .array(
+                        staticObjcXCFrameworksLinkedByDynamicXCFrameworkDependencies.flatMap { xcframework -> [String] in
+                            guard let moduleMap = xcframework.path.glob("**/module.modulemap").first
+                            else { return [] }
+                            return [
+                                "\"$(SRCROOT)/\(moduleMap.parentDirectory.relative(to: project.path).pathString)\"",
+                            ]
+                        }
+                    ),
+                ]
+            }
+        }
+
+        return (
+            try propagateSettings(graph: graph, settings: settingsToPropagate),
+            sideEffects,
+            environment
+        )
+    }
+
+    private func moduleMapFlag(
+        for xcframework: GraphDependency.XCFramework,
+        derivedDirectory: AbsolutePath,
+        project: Project
+    ) -> String {
+        let name = xcframework.path.basenameWithoutExt
+        return "-fmodule-map-file=\"$(SRCROOT)/\(derivedDirectory.appending(components: name, "Headers", "module.modulemap").relative(to: project.path).pathString)\""
```

Comment:
Am I missing something, or will this break if we have to do this for more than one dependency in the graph? Should we create sub-directories under `Headers` to scope `module.modulemap` for each XCFramework?

Reply:
This is already scoped per xcframework